src/main/java/org/neocoder/AuthService.java

```diff
@@ -0,0 +1,14 @@
+package org.neocoder;
+
+import org.mindrot.jbcrypt.BCrypt;
+
+public class AuthService {
+
+    public static String hashPassword(String password){
+        return BCrypt.hashpw(password, BCrypt.gensalt());
+    }
+
+    public static boolean comparePassword(String password, String hashedpassword){
```

Comment:
You should use always javadoc to documentation (https://www.baeldung.com/javadoc)